fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -491,9 +488,43 @@ class ShardManager {
 		metadataShard = std::make_shared<PhysicalShard>(db, "kvs-metadata");
 		physicalShards["kvs-metadata"] = metadataShard;
 		metadataShard->init();
+		writeBatch = std::make_unique<rocksdb::WriteBatch>();
 		return status;
 	}
 
+	DataShard* getDataShard(KeyRef key) { return dataShardMap.rangeContaining(key).value(); }
+
+	PhysicalShard* addRange(KeyRange range, std::string id) {
```

Comment:
nit: It would be great if we can separate the declaration of the class and the definition of its member functions, it will be much easier to scan a class without going into the details.

Reply:
Yeah, it's better to put them in different files. This file is pretty messy right now, we could do some clean up later.